bittensor/__init__.py

```diff
@@ -21,7 +21,7 @@
 
 # Bittensor code and protocol version.
 __version__ = '1.5.0'
-__version_as_int__ = (100 * 1) + (10 * 4) + (1 * 0)  # Integer representation
+__version_as_int__ = (100 * 1) + (10 * 5) + (1 * 0)  # Integer representation
```

Comment:
Is there a better way to convert to int? I can definitely see the next person updating the version forgetting to update the version on the second line.

Maybe let's do something like 

```
version_split = __version__.split(".")
__version_as_int__ = (100 * int(version_split[0])) + (10 * int(version_split[1])) + (1 * int(version_split[2]))
```

Also, I don't see this being used anywhere, so maybe better to just take this line out